packages/prettier-config/legacy.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc.json",
+  "arrowParens": "avoid",
+  "endOfLine": "auto",
+  "printWidth": 120,
+  "quoteProps": "preserve",
+  "tabWidth": 4,
+  "trailingComma": "none",
```

Comment:
I prefer the trailing comma because I'm used it but I can see why it would be preferable when diffing changes.

Reply:
@turnertime 

Not sure I understand your comment. The no-trailing-comma is the legacy setting, i.e. what we are using now (pre-this-PR). This configuration will remove all trailing commas. The new recommended setting switches to always-add-trailing-commas. This might look a bit weird initially, but I gave it a try and didn't mind it. And it is really nice when e.g. reordering lines, diffing, merging changes, etc.

Reply:
Sorry I meant to say `I prefer no trailing comma...` but I agree its worth including for diffing, ... and likely just need to get used to it.